pytest-wdl-tests/tests/literals/data/map_literals_out

```diff
@@ -0,0 +1,6 @@
+INTS
+1	10
+2	11
+MIXED
+a	1
+b	2
```

Comment:
I thought you decided that you wanted to assert that there would be a newline added here?

Reply:
I put the newline in that WDL command so that Cromwell output would match this output file. If you're referring to lack of new line at the end of file, I don't know what's right, I just wanted write_map() to give a similar format to array's write_lines()